utils/utils.py

```diff
@@ -166,19 +171,26 @@ def try_command(command):
                               stderr=subprocess.DEVNULL)
         return True
     except subprocess.CalledProcessError:
+        logging.warn(format_last_exception())
         logging.warn("No command found: %s" % (str(command)))
         return False
 
 
 def scan(command, env=None, allowed_return_codes=[]):
     try:
-        response = subprocess.check_output(command, shell=False, env=env)
+        response = subprocess.check_output(
+            command,
+            stderr=subprocess.STDOUT,
+            shell=False, env=env
+        )
```

Comment:
This improves the local error message / stacktrace handling for local scan execution where shelling out is involved.